clang/test/Analysis/solver-sym-simplification-on-assumption.c

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_analyze_cc1 %s \
+// RUN:   -analyzer-checker=debug.ExprInspection \
+// RUN:   -verify
```

Comment:
I think you need to pin the target triple as in the test you have an assumption about the size of int, as the upperbound of a value range.

Reply:
Thanks for reminding of this! Fixed!